src/base.jl

```diff
@@ -344,6 +344,20 @@ function set_unit_system!(component::Component, settings::SystemUnitsSettings)
     return
 end
 
+"""
+Sets the units base for the getter functions on the devices. It modifies the behavior of all getter functions
+"""
+function set_units_base_system!(system::System, settings::String)
+    system.units_settings.unit_system = UNIT_SYSTEM_MAPPING[settings]
```

Comment:
Should we log this?  and the one below?

Reply:
Should I add an @info statement or @debug?

Reply:
I'm imagining someone filing an issue about getting incorrect units and us wanting to know whether they set this.  Having it in the log would solve that problem.  info unless it will be called a lot, which should not be the case.